### DISCUSSION TITLE
The plug-in XML does not support wildcards. *

### DISCUSSION THREAD

**Discussion by lansgg** [apache/skywalking#10541] [Q&A]
apm-customize-enhance-plugin.jar The plug-in XML does not support wildcards. * in version 8.14 agent ?

        <method method="mikasa(java.lang.Object)" static="true"/>

        <method method=".*sa()" static="true"/>
        <method method=".*(.*)" static="true"/>

please .

**Comment by wu-sheng** [comment]
I can't see why should we. Usually, people would complain about the agent slowing down the booting duration, and if this kind of matching is added, it will be much worse. 
If you want to add, customing codes is easy.
